File: HW1/prob2_1.cc
#include<iostream>
using namespace std;

int sorting(int arr[],int l) {
	int temp; int i;
	temp=0;
	for(i=1; i<l;i++)
		if(arr[0]<arr[i]) {
			temp=arr[0];
			arr[0]=arr[i];
			arr[i]=temp;
		}
	cout<<arr[0]<<endl;
}

int main() {
	int array[]={10,34,87,3,89,23,45,67,190,78,56,90,21,45,87,43,90};
	int l=sizeof(array)/sizeof(array[0]);
	sorting(array,l);
}

File: HW1/prob2_1.py
import numpy as np
array=[10,32,2,54,100,24,1,33]
p=len(array)

def sorting(arr,i):
    for j in range(i+1,p):
        if arr[i] < arr[j]:
            arr[i],arr[j]=arr[j],arr[i]
    arr_1=arr
    return arr_1

n=int(input("Give the value of n: ")) 
semisort=array
for i in range(n):
    semisort=sorting(semisort,i)
#semisort=sorting(semisort,0)
for i in range(n):
    print(semisort[i])

